src/livekit/MediaDevicesContext.tsx

```diff
@@ -120,10 +123,17 @@ interface Props {
 }
 
 export const MediaDevicesProvider: FC<Props> = ({ children }) => {
-  // Counts the number of callers currently using device names
+  // Counts the number of callers currently using device names.
   const [numCallersUsingNames, setNumCallersUsingNames] = useState(0);
   const usingNames = numCallersUsingNames > 0;
 
+  // Use output device names for output devices on all platforms except FF.
```

Comment:
I'm not sure this comment is really telling me anything that isn't apparent from the next line: the main question would be 'why'?